### PR TITLE
feat(InfiniteGrid): add getItem, updateItem, updateItems method

### PR DESCRIPTION
**Pull request by younkue**
## Issue

#241

### CODE REVIEW

File: src/InfiniteGrid.js [hunk 1]
```diff
@@ -703,6 +703,47 @@ class InfiniteGrid extends Component {
...
+		if (itemIndex == null && typeof groupIndex === "object") {
+			const items = this.getItems();
+			const length = items.length;
+
+			for (let i = 0; i < length; ++i) {
+				if (items[i].el === groupIndex) {
+					return items[i];
+				}
+			}
+			return undefined;
```
Comment (jongmoon) [inline src/InfiniteGrid.js]: @younkue How about following?
```js
return items.find(item => item.el === groupIndex);
```

Is it not transformed by babel?

File: test/unit/InfiniteGrid.spec.js [hunk 2]
```diff
@@ -1431,6 +1431,114 @@ describe("InfiniteGrid Test", function () {
...
+      // change size
+      item1.el.style.height = `${parseInt(item1.el.style.height) + 200}px`;
+      item2.el.style.height = `${parseInt(item2.el.style.height) + 200}px`;
+
+
+      const height1 = item1.size.height;
+      const height2 = item2.size.height;
+
+      this.inst.updateItem(item1.el);
+      this.inst.updateItem(0, 1);
```
Comment (jongmoon) [inline test/unit/InfiniteGrid.spec.js]: @younkue How about only calling `updateItem` to one item.

File: src/InfiniteGrid.js [hunk 3]
```diff
@@ -703,6 +703,47 @@ class InfiniteGrid extends Component {
...
+			const group = this._items.getData(groupIndex);
+
+			return group && group.items[itemIndex || 0];
+		}
+	}
+	_updateItem(item) {
+		if (item && item.el) {
+			item.content = item.el.innerHTML;
+
+			!this.options.isEqualSize && resetSize(item);
```
Comment (jongmoon) [inline src/InfiniteGrid.js]: @younkue Why do not call resetSize when `isEqualSize`

File: src/InfiniteGrid.js (hunk 1 above)
Comment (younkue) [reply src/InfiniteGrid.js]: @jongmoon  Maybe? It seems that you need to use polyfill to use find.

File: src/InfiniteGrid.js (hunk 3 above)
Comment (younkue) [reply src/InfiniteGrid.js]: @jongmoon  All item is equal size.

**Pull request merged**